app/Models/User.php

```diff
@@ -3,16 +3,38 @@
 namespace App\Models;
 
 use App\DB\Model;
+use Exception;
+use PDO;
+use App\Core\DB;
 
 class User extends Model
 {
-    public static function add( array $params ):int
+    public static function add(array $params): int
     {
         return 0;
     }
 
-    public static function remove( int $id ):bool
+    public static function remove(int $id): bool
     {
         return true;
     }
+
+    public static function getUserById(int $id): object
+    {
+        $db       = DB::DB();
+        $query    = "SELECT id, email FROM user WHERE id= :id";
+        $prepared = $db->prepare($query);
+        $prepared->bindParam(":id", $id, PDO::PARAM_INT);
+        $prepared->execute();
+
+        $result = $prepared->fetchObject();
+
+        if ($result) {
+            return $result;
+        }
+
+        throw new Exception("User not found by given id");
```

Comment:
yaxshi olar ki, exceptionlar function-in returndan ustde olsun hemishe. Meselen, eger `user` tapilmayibsa exception atsin, eks halda return etme koduna girsin. Cunki, gelecekde ola biler ki, bir nece shert uzre yoxlama aparaq. Meselen, user tapildi amma id-i 0-dan kicikdi, bu halda ikinci bir exception atmaq istesek senin yazdigin versiyada mumkun olmayacaq. 

Qisacasi, meslehetdi ki, funksiyanin sonunda her zaman `return` qalsin. butun yoxlamalar hemin `return`-a qeder kecmish olsun.

Reply:
Məntiqlidir heç orasını düşünməmişdim) düzəldirəm indi